db/schema.rb

```diff
@@ -10,13 +10,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_08_26_202539) do
+ActiveRecord::Schema.define(version: 2022_10_17_163325) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
@jazairi can you revert this file locally and rerun your migrations with your dev env set to sqlite instead of pg? I'm curious if some of these changes are introduced due to your temp setup with this work requiring staging data.

Reply:
Yeah, that appears to be what happened. Thanks for the catch. Just pushed up a commit addressing that.